dnSpy/dnSpy/Documents/TreeView/DocumentTreeView.cs

```diff
@@ -844,6 +845,38 @@ sealed class AddDocumentInfo {
 			}
 		}
 
+		static void ResolveInternetShortcutFiles(string[] filenames) {
+			for (var i = 0; i < filenames.Length; i++) {
+				var filename = filenames[i];
+				if (!filename.EndsWith(".url", StringComparison.OrdinalIgnoreCase))
+					continue;
+				if (!File.Exists(filename))
+					continue;
+				try {
+					var urlString = GetUrlFromInternetShortcutFile(filename);
+					if (urlString == null)
+						continue;
+					var urlUri = new Uri(urlString);
+					if (urlUri is { IsAbsoluteUri: true, IsFile: true }) {
+						filenames[i] = urlUri.AbsolutePath;
+					}
+				}
+				catch {
+					// ignore any unexpected parsing errors
```

Comment:
Redundant comment, the code flow is self-explanatory here.

Reply:
See previous conversation.